app/views/permissions/show.html.erb

```diff
@@ -8,7 +10,7 @@
     <p>
       <b>Collections |
         <% if @collection_options %>
-          <%= @collection_entry_ids.blank? ? 'All Collections' : "#{@collection_entry_ids.count} Selected Collections" %>
+          <%= @collection_entry_ids.blank? && @hidden_entry_titles.blank? ? 'All Collections' : "#{@collection_entry_ids.count + @hidden_entry_titles.count} Selected Collections" %>
```

Comment:
If I understand this correctly, we're increasing the collection count, and displaying the message to the user, but not listing the collections anywhere, is that right?